### PR TITLE
EIP-1: require at least one github username

### PR DESCRIPTION
**Pull request by axic**
This has been discussed a few times. The main problem is that editors have a challenge finding EIP authors and getting their response, unless there is a clear GitHub username. This is blocking a lot of outstanding change requests.

Awaiting feedback, @Souptacular @nicksavers @MicahZoltu @lightclient

### CODE REVIEW

File: EIPS/eip-1.md [hunk 1]
```diff
@@ -226,23 +230,15 @@ If you are interested in assuming ownership of an EIP, send a message asking to
...
-` * Nick Savers (@nicksavers)`
-
-` * Martin Becze (@wanderer)`
-
-` * Greg Colvin (@gcolvin)`
-
-` * Alex Beregszaszi (@axic)`
-
-` * Micah Zoltu (@MicahZoltu)`
+- Nick Johnson (@arachnid)
```
Comment (axic) [inline EIPS/eip-1.md:233]: I can remove this commit, it is only about making this more readable, not sure what was original motivation for the previous version.

File: EIPS/eip-1.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ status: Active
 type: Meta
 author: Martin Becze <mb@ethereum.org>, Hudson Jameson <hudson@ethereum.org>, et al.
 created: 2015-10-27
-updated: 2015-12-07, 2016-02-01, 2018-03-21, 2018-05-29, 2018-10-17, 2019-05-19, 2019-12-04, 2020-06-17
+updated: 2015-12-07, 2016-02-01, 2018-03-21, 2018-05-29, 2018-10-17, 2019-05-19, 2019-12-04, 2020-06-17, 2020-09-10
```
Comment (MicahZoltu) [inline EIPS/eip-1.md:8]: Isn't this information already in GitHub?  What is the purpose of an ever growing list of dates here?

File: EIPS/eip-1.md [hunk 3]
```diff
@@ -290,6 +286,8 @@ Dec 4, 2019: The "Security Considerations" section was introduced.
 
 June 17, 2020: Canonicalizes the format for referencing EIPs by number in the "Style Guide".
 
+Sep 10, 2020: Require the `author` header to include at least one GitHub username.
+
```
Comment (MicahZoltu) [inline EIPS/eip-1.md:290]: This also feels redundant with git commit history.  Why aren't we just letting our version control system do what version control systems do best (track change histories)?

**Review by MicahZoltu — APPROVED** [review]
My feedback is all around EIP-1 conventions, nothing wrong with this particular PR.

File: EIPS/eip-1.md (hunk 2 above)
Comment (lightclient) [reply EIPS/eip-1.md:8]: I agree here, it sure causes the front matter to look really cluttered.

**Review by lightclient — APPROVED** [review]

**Pull request merged**